go/enclave/components/batch_executor.go

```diff
@@ -220,7 +221,7 @@ func (executor *batchExecutor) ComputeBatch(context *BatchExecutionContext, fail
 	}
 
 	// the logs and receipts produced by the EVM have the wrong hash which must be adjusted
-	for _, receipt := range txReceipts {
+	for _, receipt := range allReceipts {
```

Comment:
qq. Was this change required to fix the bug?

Reply:
Not strictly. This was required to make sure the message bus contract address hadn't changed when I was debugging the issue.
Right now everywhere that we refer to the message bus contract address it's hardcoded, which is kinda ok if you know ahead of time, otherwise you'll be looping around trying to find the address output somewhere like I did.
Left it there because I think it's better to be dynamic and potentially it enables Matt's info endpoint and allows Moray to read the address from the logs.

I understand that we were not initially showing the synthetic tx receipts, don't know if there was a major reason behind it, can revert back to the old behavior if this is no buenos!